requirements.txt

```diff
@@ -5,3 +5,4 @@ djangorestframework==3.9.0
 PyJWT==1.7.1
 six==1.12.0 
 psycopg2-binary==2.7.6.1
+pep8==1.7.1
```

Comment:
pep8 has been renamed to `pycodestyle`.
Use of the pep8 tool will be removed in a future release.
Please install and use `pycodestyle` instead.